src/components/BookList.js

```diff
@@ -1,29 +1,84 @@
-import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
+import axios from 'axios';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchBooks, removeBook } from '../redux/books/booksSlice';
 import CreateBook from './CreateBook';
-import { removeBook } from '../redux/books/booksSlice';
 
-// BookList.js
-
-// ...
+const API_ID = 'u3BFViY5zE36mpKmwqF4';
+const API_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
 
 function BookList() {
-  const books = useSelector((state) => state.books.books);
   const dispatch = useDispatch();
+  const library = useSelector((state) => state.books.books);
+  // const books = Object.values(library).reduce((acc, books) => acc.concat(books), []);
```

Comment:
- [ ] Great job on implementing the functionality based on the project requirement but I will kindly suggest you remove the commented code line from your code base This makes your code look so messy and look distractive to another developer when reading your code please kindly fix it .